cli/template/extras/src/server/db/db-prisma-planetscale.ts

```diff
@@ -0,0 +1,21 @@
+import { Client } from "@planetscale/database";
+import { PrismaPlanetScale } from "@prisma/adapter-planetscale";
+import { PrismaClient } from "@prisma/client/edge";
```

Comment:
Currently, the edge client doesn't support custom adapters (see https://github.com/prisma/prisma/issues/21394), so this would need to be the normal prisma client until that issue is resolved. Once resolved, this can change (and likely also remove some of the global fluff since connection pooling is technically unnecessary when using the edge stuff).

Reply:
Didn't know that, thank you!